docs/getting-started/release-notes.md

```diff
@@ -11,9 +11,14 @@ The last stable version is v2.2.0. To see it select versions: 'stable' in the le
 ### Current changes
 Current changes from stable version are noted here.
 
+**Modified objects:**
 * [StructuralLoadCase](../loads/structuralloadcase.md)
    * From enum "Load type" value "Standard" was deleted. It was not possible to differentiate between "Standard" and "Others". What was exported as "Standard" should now be exported as "Others".
 
+* [StructuralSurfaceActionFree](../loads/structuralsurfaceactionfree.md)
+   * enum "Coordinate system" extended with value "Member LCS"
+   * new properties "Validity", "Validity to \[m]", "Validity from \[m]", and "Local Z directionZ" added
```

Comment:
Typo at the end of  "Local Z direction**Z**"

Reply:
done 👍